db_stress_tool/db_stress_test_base.cc

```diff
@@ -2528,19 +2530,17 @@ void StressTest::TestCompactRange(ThreadState* thread, int64_t rand_key,
                         << cro.blob_garbage_collection_age_cutoff;
   Status status = db_->CompactRange(cro, column_family, &start_key, &end_key);
 
-  if (!status.ok() && !status.IsManualCompactionPaused() &&
-      !std::strstr(status.getState(), "injected") &&
-      !(status.IsNotSupported() &&
-        (std::strstr(status.getState(), "another thread is refitting") ||
-         std::strstr(
-             status.getState(),
-             "Levels between source and target are not empty for a move")))) {
+  // TOOD (hx235): allow an exact list of tolerable failures under stress test
+  bool non_ok_status_allowed = status.IsManualCompactionPaused() ||
+                               std::strstr(status.getState(), "injected") ||
+                               status.IsInvalidArgument() ||
+                               status.IsNotSupported();
+  if (!status.ok() && !non_ok_status_allowed) {
     fprintf(stdout,
             "Unable to perform CompactRange(): %s under specified "
             "CompactRangeOptions: %s (Empty string or "
             "missing field indicates default option or value is used)\n",
             status.ToString().c_str(), compact_range_opt_oss.str().c_str());
-    thread->stats.AddErrors(1);
     // Fail fast to preserve the DB state.
     thread->shared->SetVerificationFailure();
   }
```

Comment:
Print those allowed non-ok status to stdout?

Reply:
Sure

Reply:
Fixed